### ISSUE TITLE
Error to update Microsoft.Data.SqlClient 5.1.4

### ISSUE THREAD

**Issue by jvsancho80** [dotnet/SqlClient#2295]
When update this version I have this error. But if I use de preview 5.2.0-preview3.23201.1 works ok

![image](https://github.com/dotnet/SqlClient/assets/94198387/a2e3e1c3-5ec0-480e-8f8f-5656e0bcc932)

**Comment by ErikEJ** [comment]
@jvsancho80 Correct, this is fixed in 5.2 preview 5 only, due by the end of this month

**Comment by jvsancho80** [comment]
Ok @ErikEJ  . Thanks so much

**Comment by jvsancho80** [comment]
@ErikEJ. Another preview?  When publish 5.2?

**Comment by ErikEJ** [comment]
@jvsancho80 Plan is 5.2 at end of Feb

**Comment by jvsancho80** [comment]
Ok. Thanks you @ErikEJ

**Comment by JRahnama** [comment]
also, preview5 is scheduled toward end of this month.

**Comment by kf-gonzalez** [comment]
Closing Issue, being addressed by Preview5
